README.md

```diff
@@ -307,6 +307,9 @@ Metadata flags:
 Shared memory flags:
 - :whale: `--shm-size`: Size of `/dev/shm`
 
+GPU flags:
+- :whale: `--gpus`: GPU devices to add to the container ('all' to pass all GPUs). `nvidia-container-cli` is needed.
```

Comment:
Can we have `./docs/gpu.md` to explain all the options?

We should also clarify how to set up GPU for rootless. (Can be another PR)